vars/preCommitToJunit.groovy

```diff
@@ -42,16 +43,16 @@ def call(Map args = [:]) {
     }
   }
   if (inprogress) {
-    data += toJunit(id, status, message)
+    data += toJunit(id, status, message, reportSkipped)
   }
   data += '</testsuite>'
 
   writeFile file: output, text: data, encoding: 'UTF-8'
 }
 
-def toJunit(String name, String status, String message) {
+def toJunit(String name, String status, String message, reportSkipped=false) {
```

Comment:
Is it redundant to set this to false, as the caller already sets the default value to false?

Reply:
I wanted to play safe, in case someone uses this function, theoretically it's something internal